gubernator/log_parser.py

```diff
@@ -59,7 +59,7 @@ def log_html(lines, matched_lines, hilight_words, skip_fmt):
             skip_string = ('<span class="skip">'
                 '<a href="javascript:show_skipped(\'%s\')" onclick="this.style.display=\'none\'">'
                 '%s</a></span>'
-                '<span class="skipped" id=%s style="display:none;">') % (skip_id,
+                '<span class="skipped" id="%s" style="display:none; float: left;">') % (skip_id,
```

Comment:
In my experience floating elements have lots of nasty corner cases. I think if you change the style to inline-block in show_skipped instead, it should fix your problem.